plugins/woocommerce/templates/global/quantity-input.php

```diff
@@ -23,29 +23,43 @@
 		<input type="hidden" id="<?php echo esc_attr( $input_id ); ?>" class="qty" name="<?php echo esc_attr( $input_name ); ?>" value="<?php echo esc_attr( $min_value ); ?>" />
 	</div>
 	<?php
-} else {
-	/* translators: %s: Quantity. */
-	$label = ! empty( $args['product_name'] ) ? sprintf( esc_html__( '%s quantity', 'woocommerce' ), wp_strip_all_tags( $args['product_name'] ) ) : esc_html__( 'Quantity', 'woocommerce' );
+}
+/* translators: %s: Quantity. */
+$label = ! empty( $args['product_name'] ) ? sprintf( esc_html__( '%s quantity', 'woocommerce' ), wp_strip_all_tags( $args['product_name'] ) ) : esc_html__( 'Quantity', 'woocommerce' );
+?>
+<div class="quantity">
+	<?php
+	/**
+	 * Hook to output something before quantity input field
+	 *
+	 * @since 3.6.0
```

Comment:
```suggestion
	 * Hook to output something before the quantity input field.
	 *
	 * @since 7.0.0
```

(Unless I'm mistaken, this hook is newly introduced, so the version tag should reflect that. Sidenote: at time of writing GH is highlighting and transforming `@since` in a weird way ... but I believe that's presentational only.)